app/utils/helpers.js

```diff
@@ -0,0 +1,16 @@
+
+export const normalizeUrl = (url) => {
```

Comment:
I would recommend you for those kinds of helpers create tests with all possible values you can expect and have.